src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -529,6 +533,14 @@
     <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.PublicIpAddressTests\TestPublicIpStandardSku.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+<!-- AAI - Something needs to be done here but i dont understand what
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.PublicIpAddressTests\TestPublicIpAddressCRUDPublicIpPrefix.json">
```

Comment:
seems a total of three tests got added, you'll have to add the three recorded files for the tests.

Reply:
Ah got it!

Reply:
Updated